src/main/java/com/manardenza/console/ContentsUserMenu.java

```diff
@@ -0,0 +1,346 @@
+
+    private HotelController hotelController;
+    private ReservationController reservationController;
+    private UserController userController;
+    private List listData;
+
+    public ContentsUserMenu(HotelController hotelController,
+                            ReservationController reservationController, UserController userController) {
+        this.hotelController = hotelController;
+        this.reservationController = reservationController;
+        this.userController = userController;
+    }
+
+    public void mainMenu() throws IOException {
+        outputSplitLine();
+        outputDataInConsole("\t\tWelcome to manardenza.com");
+        outputSplitLine();
+        outputDataInConsole("\n\tAuthorization");
+        showUserMenu("Main menu");
+        int actionMain = readerMenuFromConsole("Choose action: ");
+        switch (actionMain) {
+            case (1):
+                registrationUserMenu();
+                break;
+            case (2):
+                outputDataInConsole("\tBuy!");
+                System.exit(0);
+                break;
+            default:
+                outputDataInConsole("Incorrect menu item");
+                mainMenu();
+                break;
+        }
+    }
+
+    public void serviceMenu() throws IOException {
+        boolean flagMenu = true;
+        while (flagMenu) {
+            outputSplitLine();
+            outputDataInConsole("\n\tService menu");
+            showUserMenu("Service menu");
+            int actionService = readerMenuFromConsole("Choose action: ");
+            switch (actionService) {
+                case (1):
+                    outputSplitLine();
+                    outputDataInConsole("For find hotel by name complete the following form");
+                    findHotelByNameMenu();
+                    outputSplitLine();
+                    break;
+                case (2):
+                    outputSplitLine();
+                    outputDataInConsole("For find hotel by city complete the following form");
+                    outputDataInConsole(findHotelByCityMenu().toString());
+                    outputSplitLine();
+                    break;
+                case (3):
+                    outputSplitLine();
+                    outputDataInConsole("To find the room fill in the following form");
+                    findRoomMenu();
+                    outputSplitLine();
+                    break;
+                case (4):
+                    outputSplitLine();
+                    outputDataInConsole("Your reserved rooms: ");
+                    getAllUserReservationsMenu();
+                    break;
+                case (5):
+                    outputSplitLine();
+                    outputDataInConsole("To cancel a reservation of the room, fill in the following form");
+                    cancelReservationMenu();
+                    outputSplitLine();
+                    break;
+                case (6):
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png 'Severity: MAJOR') Reduce this switch case number of lines from 6 to at most 5, for example by extracting code into methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1151)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove those useless parentheses. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessParenthesesCheck)